firebase-installations/src/main/java/com/google/firebase/installations/local/PersistedFidEntry.java

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.installations.local;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.google.auto.value.AutoValue;
+
+/**
+ * This class represents a persisted fid entry in {@link PersistedFid}, which contains a few
+ * Firebase Installation attributes and the persisted status of this entry.
+ */
+@AutoValue
+public abstract class PersistedFidEntry {
```

Comment:
Please feel free to ignore this comment.
Maybe, it's easier to discuss in person and just submit this PR with this name.
I'm just wondering:
Why can't we just call this "FirebaseInstallation"?
(or "PersistedFirebaseInstallation" in case you want to make it obvious that it's received from local storage)
Let's discuss in person after this is submitted.